docs/user-manual/en/amqp-broker-connections.md

```diff
@@ -163,7 +161,29 @@ For more information refer to the "brokered messaging" documentation for [Apache
 
 *Important:* You do not need to configure the router with a connector or auto-links to communicate with the broker. The brokers peer configuration replaces these aspects of the router waypoint usage.
 
-# Mirror 
+## Address Consideration
+It is highly recommended that you keep `address name` and `queue name` the same, as when you use a queue with its distinct name (as in the following example), senders and receivers will always use the `address name` when creating the remote endpoint.
+
+```xml
+<broker-connections>
+    <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker">
+       <sender address-match="queues.#"/>
+    </amqp-connection>
+</broker-connections>
+<addresses>
+     <address name="queues.A">
+        <anycast>
+           <queue name="distinctNameQueue.A"/>
+        </anycast>
+     </address>
+</addresses>
+```
+
+In the above example the `broker connection` would create an AMQP sender towards "queues.A". 
```

Comment:
For clarity I would use "addressprefix.#" and  "addressprefix.A"or some such alterantive. Its just simpler to consider when not having "queues" in the address value when thinking of it as an address name.